Sources/scipio/PrepareCommands.swift

```diff
@@ -20,9 +20,22 @@ extension Scipio {
         var packageDirectory: URL = URL(fileURLWithPath: ".")
 
         @Option(name: [.customLong("cache-policy")],
-                help: "Specify how to reuse cache. (\(CachePolicy.allCases.map(\.rawValue).joined(separator: " / ")))",
+                help: """
+                      [DEPRECATED] Use --framework-cache-policy instead.
+                      Specify how to reuse framework cache. (\(CachePolicy.allCases.map(\.rawValue).joined(separator: " / ")))
+                      """,
                 completion: .list(CachePolicy.allCases.map(\.rawValue)))
-        var cachePolicy: CachePolicy = .project
+        var cachePolicy: CachePolicy?
+
+        @Option(name: [.customLong("framework-cache-policy")],
+                help: "Specify how to reuse framework cache. (\(CachePolicy.allCases.map(\.rawValue).joined(separator: " / ")))",
+                completion: .list(CachePolicy.allCases.map(\.rawValue)))
+        var frameworkCachePolicy: CachePolicy?
+
+        @Option(name: [.customLong("resolved-packages-cache-policy")],
+                help: "Specify how to reuse resolved packages cache. (\(CachePolicy.allCases.map(\.rawValue).joined(separator: " / ")))",
+                completion: .list(CachePolicy.allCases.map(\.rawValue)))
+        var resolvedPackagesCachePolicy: CachePolicy?
```

Comment:
ditto

Reply:
We consider resolvedPackagesCachePolicy to be necessary only for large projects, so the default is intentionally set to nil.

Reply:
Oh sorry, since CachePolicy has a `.disabled` case, it seems that setting the default value (`.disabled`) was the correct choice instead of nil.